test/Table_Tests/data/windows.csv

```diff
@@ -0,0 +1,2 @@
+a,b,c

```

Comment:
Probably worth this file having Windows file endings too

Reply:
It had on my machine, but maybe the autocrlf settings messed up.

Anyway - the line endings logic is handled by the test which generates files in the run-time to ensure that all kinds of endings are tested regardless of the git settings. Here this depends on git settings too much.